frontend/src/components/ThemedButtons/ThemedButton.tsx

```diff
@@ -28,16 +31,27 @@ function ThemedButton({
 		disabled: ariaDisabled,
 	});
 
+	const tooltipId = `themed-button-desc-${tooltip?.id
```

Comment:
Somewhat confused by why the tooltip div needs an id.

Reply:
I think we may have covered this in a daily, but for prosperity's sake, this ID links the `aria-describedby` on the `button` to the tooltip `div`.